src/common/components/Alert.tsx

```diff
@@ -0,0 +1,43 @@
+import { PropsWithChildren } from 'react';
+import classes from './Alert.module.scss';
+
+export type Variant = 'info' | 'error';
+
+export type InfoAlertProps = PropsWithChildren<{
+  variant: Variant;
+  title?: string;
+  message?: string;
+}>;
```

Comment:
Can you tell I'm a bootstrap lover? Variant is from the design language of react-bootstrap, and I tend to bring it everywhere, but this may be moot if using mui alert instead. I'll explore that and get back to this here.